internal/services/mssql/mssql_server_dns_alias_resource.go

```diff
@@ -0,0 +1,143 @@
+package mssql
+
+import (
+	"context"
+	"fmt"
+	"time"
+
+	"github.com/hashicorp/terraform-plugin-sdk/v2/helper/schema"
+	"github.com/hashicorp/terraform-provider-azurerm/helpers/azure"
+	"github.com/hashicorp/terraform-provider-azurerm/internal/sdk"
+	"github.com/hashicorp/terraform-provider-azurerm/internal/services/mssql/parse"
+	"github.com/hashicorp/terraform-provider-azurerm/internal/services/mssql/validate"
+	"github.com/hashicorp/terraform-provider-azurerm/internal/tf/pluginsdk"
+	"github.com/hashicorp/terraform-provider-azurerm/utils"
+)
+
+type ServerDNSAliasModel struct {
+	ServerName    string `tfschema:"server_name"`
+	ResourceGroup string `tfschema:"resource_group_name"`
```

Comment:
Normally we use `mysql_server_id` here instead of `server_name` + `resource_group_name`. As the latter twos can be parsed from the `mysql_server_id`.

Reply:
this resource is for mssql, do you mean use a mssql_server_id field to replace these two fields?

Reply:
Yes, sorry for the mis reading 🙈